test/algorithms/buffer/buffer_multi_linestring.cpp

```diff
@@ -212,6 +212,19 @@ void test_all()
     }
 #endif
 
+    {
+        // This issue was detected for CCW order and only CW is tested by default.
```

Comment:
Right... this is actually not the intention, but with this comment it's fine for now.